src/Helper/HeadScript.php

```diff
@@ -53,15 +53,17 @@
 class HeadScript extends AbstractStandalone
 {
     /**
-     * Script type constants
-     *
-     * @const string
+     * @deprecated Since 2.40.0 This constant will be removed in 3.0
      */
     public const FILE   = 'FILE';
```

Comment:
@gsteel 
Can you check this?

Reply:
Not sure I understand - the constant is gone in 3.0

Reply:
Sorry, I mean the failure message:

> Equals sign not aligned correctly; expected 1 space but found 3 spaces

Reply:
🤦‍♂️ Sorry - Just a sec!